src/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -146,6 +179,144 @@ private bool TryParseArguments(out SubCommand subCommand, out string parameter)
             }
         }
 
+        private async Task GetStageDetails(StageManagementResource stageManagementResource, string stageId)

```

Comment:
Could we call this PrintStageDetails? it doesn't seem to actually return anything?